<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration</class>
 <widget class="QWidget" name="registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registeration</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>42</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>65</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>130</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
